app/Enums/QuestionOptionType.php

```diff
@@ -16,9 +16,9 @@ public function display(): string
     {
         return match ($this) {
             QuestionOptionType::OPEN => 'Open antwoord',
-            QuestionOptionType::IMAGE => 'Afbeelding',
-            QuestionOptionType::VIDEO => 'Video',
-            QuestionOptionType::VOICE => 'Spraakopname',
+            QuestionOptionType::IMAGE => 'Afbeeldingen',
+            QuestionOptionType::VIDEO => "Video's",
```

Comment:
```suggestion
            QuestionOptionType::VIDEO => 'Video\'s',
```

Anders krijg je wellicht problemen met het laten zien omdat je zo ook normale strings doet weergeven.